modules/build/src/main/scala/scala/build/options/BuildOptions.scala

```diff
@@ -1,9 +1,9 @@
 package scala.build.options
-
```

Comment:
Why remove this line?

Reply:
you mean the empty line? most probably it got removed by either scalafmt or scalafix